cmd/snap-bootstrap/cmd_initramfs_mounts_run_test.go

```diff
@@ -928,6 +939,170 @@ func (s *initramfsMountsSuite) TestInitramfsMountsRunModeEncryptedDataHappy(c *C
 
 	c.Assert(filepath.Join(dirs.SnapBootstrapRunDir, "secboot-epoch-measured"), testutil.FilePresent)
 	c.Assert(filepath.Join(dirs.SnapBootstrapRunDir, "run-model-measured"), testutil.FilePresent)
+
+	checkDegradedJSON(c, "unlocked.json", map[string]interface{}{
+		"ubuntu-boot": map[string]interface{}{},
+		"ubuntu-data": map[string]interface{}{
+			"find-state":   "found",
+			"unlock-state": "unlocked",
+			"unlock-key":   "run",
+		},
+		"ubuntu-save": map[string]interface{}{
+			"find-state":   "found",
+			"unlock-state": "unlocked",
+			"unlock-key":   "run",
+		},
+		"error-log": interface{}(nil),
+	})
```

Comment:
@pedronis  For the run mode, I do not store the mount state. Only unlock state. Because I do not think we need it. Is that OK not to make it complete?

Reply:
Also `"ubuntu-boot"` is empty because we do not need it either.

I could make those files pointers so that the `omitempty` actually works and we have a nil instead.

Reply:
it's ok as long as we document it somewhere in the code, maybe in the struct itself?